tests/lib/prepare-restore.sh

```diff
@@ -654,6 +670,19 @@ restore_suite_each() {
     # restore test directory saved during prepare
     tests.backup restore
 
+    # Save all the installed packages and remove the new packages installed 
+    if os.query is-classic; then
+        distro_get_installed_packages > installed-final.pkgs
+        diff -u installed-initial.pkgs installed-final.pkgs | grep -E "^\+" | tail -n+2 | cut -c 2- > installed-new.pkgs
+
+        # shellcheck disable=SC2002
+        packages="$(cat test | tr "\n" " ")"
+        if [ -n "$packages" ]; then
+            # shellcheck disable=SC2086
+            distro_purge_package $packages
+        fi
```

Comment:
should we also remove the `installed-initial.pkgs` and `installed-final.pkgs` files, or is it not important?

Reply:
thanks for reviewing, I prefer to keep it so in case of an error I can check the packages installed, at least at the beginning